.gitignore

```diff
@@ -11,7 +13,7 @@
 /composer.phar
 /coverage
 /node_modules/*
-/vendor/*
+/vendor/
```

Comment:
I got this from the default .gitignore in symfony, see: https://github.com/symfony/symfony/blob/2.8/.gitignore#L1